app/workers/fetch_public_posts.rb

```diff
@@ -4,7 +4,7 @@
 
 module Workers
   class FetchPublicPosts < Base
-    sidekiq_options queue: :http_service
+    sidekiq_options queue: :high
```

Comment:
I'd like to keep this `medium` then. At least personally I quite often add a profile and then refresh for the posts a little while.